lib/find-view-matches.js

```diff
@@ -57,6 +63,10 @@ function attrSelector(attr, value) {
 	].join('');
 }
 
+function prefixSelector(prefix, value) {
+	return value.replace(new RegExp('^\\'+ prefix +'?(.*)$', 'i'), function (match) { return prefix + match });
```

Comment:
@ramonvictor Sorry, lost your comment. Meant to move it here quoted since this was the updated source. 

You were asking why I am doing the regex, correct?

Short answer I like regexes, long answer is this could be done various ways but the basic logic is:

1. Check if prefix exists
2. If prefix exists return the value, else prepend the prefix and return the new value.

Could also look like this,

```js
function prefixSelector(prefix, value) {
  return value.indexOf(prefix) > -1 ? value : prefix + value;
}
```

Hindsight, this would probably be easier lol